examples/ping.ts

```diff
@@ -0,0 +1,39 @@
+import { Client, Message, Intents } from '../mod.ts'
+
+const client = new Client();
+
+client.on("ready", () => {
+    console.log(`Logged in as ${client.user?.tag}!`);
+});
+
+client.on("messageCreate", (msg: Message) => {
```

Comment:
I hope you can comment on it so that others can understand

Reply:
Isn't it pretty straightforward? Still, I can do.

Reply:
I think it will be okay if there is some